astropy/units/format/vounit.py

```diff
@@ -108,14 +105,7 @@ def _get_unit(cls, t):
 
     @classmethod
     def _parse_unit(cls, unit, detailed_exception=True):
```

Comment:
Not really on you, but a parameter that is not used at all is a bit weird. Would it be more logical to change the default to `False` and just pass it on in the `super()` call? Also fine to stick with what you have, though.

Reply:
Yes, that makes sense.